### PR TITLE
Blackburrow new item in segment code

### PR DESCRIPTION
**Pull request by Wirednuke**
Adding a new item to the segment code in black burrow. I've never done this before, this needs to be looked at closely to make sure I did it right. @jkachhad

### CODE REVIEW

**Review by Chilly021 — APPROVED** [review]

**Comment by Wirednuke** [comment]
Found a mistake and fixed it.

**Review by Chilly021 — APPROVED** [review]

**Review by Leolus — DISMISSED** [review]

**Review by sproatba — APPROVED** [review]

**Comment by Wirednuke** [comment]
I appreciate you guys taking the time to approve this. Lux informed me of a change I need to make before this can work, so I am going to have to dismiss the approvals.

**Comment by Wirednuke** [comment]
Ready for review!

**Review by Chilly021 — APPROVED** [review]

**Pull request closed without merging**
